### ISSUE TITLE
Login dialog doesn't register keyboard input until hovered over with mouse despite blinking cursor

### ISSUE THREAD

**Issue by MerkaST** [ValveSoftware/steam-for-linux#9728]
#### Your system information

* Steam client version (build number or date): 1687386907
* Distribution (e.g. Ubuntu): Lubuntu 22.04
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/11862742/steam-logs.tar.gz)
* GPU: AMD R9 380X

#### Please describe your issue in as much detail as possible:
When launching Steam, the login window will have a blinking cursor in the name field implying text can directly be entered as has been the case in the past, but currently no keyboard input appears to be registered at all until the mouse is moved over the window and a few pixels inwards. Alt-tabbing and min- and maximising doesn't change this either, if you're careful you can even grab the title bar, move the window around, and still not be able to input your credentials until you move the mouse fully into the window.

#### Steps for reproducing this issue:

1. Start Steam
2. See login dialog
3. Attempt to input credentials

**Comment by kisak-valve** [comment]
Possible duplicate of #9406.

**Comment by Zorrototo** [comment]
Indeed, I tried yesterday and I could reply to chat inside the overlay without the need to hover the mouse cursor above the text area.
